acceptance/builder_test.go

```diff
@@ -0,0 +1,248 @@
+			it("invalid group toml file", func() {
+				command := exec.Command(
+					"docker",
+					"run",
+					"--rm",
+					"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
+					"--env", "CNB_GROUP_PATH=/cnb/group_tomls/wrong_group.toml",
+					"--env", "CNB_PLAN_PATH=/cnb/plan_tomls/always_detect_plan.toml",
+					builderImage,
+				)
+				output, err := command.CombinedOutput()
+				h.AssertNotNil(t, err)
+				expected := "failed to read buildpack group: Near line"
+				h.AssertStringContains(t, string(output), expected)
+			})
+
+			// .../cmd/lifecycle/builder.go#72-L74
+			it("invalid builpack api group toml file", func() {
+				command := exec.Command(
+					"docker",
+					"run",
+					"--rm",
+					"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
+					"--env", "CNB_GROUP_PATH=/cnb/group_tomls/invalid_buildpack_api_group.toml",
+					"--env", "CNB_PLAN_PATH=/cnb/plan_tomls/always_detect_plan.toml",
+					builderImage,
+				)
+				output, err := command.CombinedOutput()
+				h.AssertNotNil(t, err)
+				expected := "failed to : parse buildpack API"
+				h.AssertStringContains(t, string(output), expected)
+			})
+		})
+
+		// .../cmd/lifecycle/builder.go#121
+		when("error during parse plan", func() {
+			it("no default plan.toml file in default location", func() {
+				command := exec.Command(
+					"docker",
+					"run",
+					"--rm",
+					"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
+					"--env", "CNB_GROUP_PATH=/cnb/group_tomls/always_detect_group.toml",
+					builderImage,
+				)
+				output, err := command.CombinedOutput()
+				h.AssertNotNil(t, err)
+				expected := "failed to parse detect plan"
+				h.AssertStringContains(t, string(output), expected)
+			})
+
+			//TODO: check some output file for this case not except any error message
+			it("empty parse plan.toml file", func() {
+				command := exec.Command(
+					"docker",
+					"run",
+					"--rm",
+					"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
+					"--env", "CNB_PLAN_PATH=/cnb/plan_tomls/empty_plan.toml",
+					"--env", "CNB_GROUP_PATH=/cnb/group_tomls/always_detect_group.toml",
+					builderImage,
+				)
+				_, err := command.CombinedOutput()
+				//print(string(output), err)
+				h.AssertNil(t, err)
+				//expected := "failed to read buildpack order file"
+				//h.AssertStringContains(t, string(output), expected)
+			})
+
+			it("invalid parse plan.toml file", func() {
+				command := exec.Command(
+					"docker",
+					"run",
+					"--rm",
+					"--env", "CNB_PLATFORM_API="+latestPlatformAPI,
+					"--env", "CNB_PLAN_PATH=/cnb/plan_tomls/wrong_plan.toml",
+					"--env", "CNB_GROUP_PATH=/cnb/group_tomls/always_detect_group.toml",
+					builderImage,
+				)
+				output, err := command.CombinedOutput()
+				h.AssertNotNil(t, err)
+				expected := "failed to parse detect plan: Near line"
+				h.AssertStringContains(t, string(output), expected)
+			})
+
+		})
+	})
+
+	when("PlaceHolders need to replace with defaults", func() {
+
+		// .../cmd/lifecycle/builder.go#48-50
```

Comment:
I think it's better to specify the `Args` function in the comment and not specific line numbers because they can change on every commit.

Reply:
okey i will change all comment lines like that

Reply:
Done